lexicon/tests/providers/test_aliyun.py

```diff
@@ -0,0 +1,17 @@
+"""test module for aliyun provider"""
+
+# Test for one implementation of the interface
+from unittest import TestCase
+from lexicon.tests.providers.integration_tests import IntegrationTests
+
+# Hook into testing framework by inheriting unittest.TestCase and reuse
+# the tests which *each and every* implementation of the interface must
+# pass, by inheritance from integration_tests.IntegrationTests
+class AliyunProviderTests(TestCase, IntegrationTests):
+    """Integration tests for Foo provider"""
+    provider_name = 'aliyun'
+    domain = 'mean.space'
+
+    """filter access key id"""
+    def _filter_query_parameters(self):
+        return ['AccessKeyId', 'Signature', 'SignatureNonce', 'Timestamp']
```

Comment:
Not mandatory at all, but nice feature if you want to keep cassettes with sanitized sensible material that are not removed, just replaced by dummy values. VCRpy allows you to declare tuples of two elements (eg. `('AccessKeyId', 'KEY_ID')` instead of just the string of the query parameter name to filter (eg. `AccessKeyId`).

In this case, instead of just removing the parameter, VCRpy will replace its value in the cassette by the second value of the tuple.

https://vcrpy.readthedocs.io/en/latest/advanced.html#advanced-use-of-filter-headers-filter-query-parameters-and-filter-post-data-parameters